site/styles/_variables.scss

```diff
@@ -120,20 +125,22 @@ $page-detail-padding-top-md: 153px !default;
 // Footer
 $footer-brand-min-width: 100px !default;
 $footer-brand-min-width-md: 100px !default;
+$footer-fs-invert-percentage: 100%; // Change to 100% for light theme
```

Comment:
I think this needs to become some sort of `theme-` variable so that people know they have to change it.

Reply:
done